optuna/storages/_rdb/alembic/versions/v2.0.0.a.py

```diff
@@ -0,0 +1,36 @@
+"""empty message
+
+Revision ID: v2.0.0.a
+Revises: v1.3.0.a
+Create Date: 2020-06-23 17:32:57.341978
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "v2.0.0.a"
+down_revision = "v1.3.0.a"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "trials",
+        sa.Column(
+            "datetime_last_update",
+            sa.DateTime(),
+            server_default=sa.text("(CURRENT_TIMESTAMP)"),
```

Comment:
In terms of portability, it is prefer to pass `sa.func.now()` at server_default. I guess `sa.text("(CURRENT_TIMESTAMP)")` doesn't properly work on some databases.

```python
from alembic import op
from sqlalchemy import Column, TIMESTAMP, func

# specify "DEFAULT NOW" along with the column add
op.add_column('account',
    Column('timestamp', TIMESTAMP, server_default=func.now())
)
```

https://alembic.sqlalchemy.org/en/latest/ops.html#alembic.operations.Operations.add_column

Reply:
Thanks for your suggestion. `CURRENT_TIMESTAMP` was automatically generated from the changes to the model using `func.now` but if we can use `func.now()` in the migration script we probably should. 

This problem remains however, https://github.com/optuna/optuna/pull/1405#discussion_r444093388 and I haven't found a proper way to address it. I'm wondering if we should allow `NULL` and populate the column manually in this migration script.

Reply:
Addressed. See https://github.com/optuna/optuna/pull/1405#discussion_r444710090.